modules/nw-ipfailover-remove.adoc

```diff
@@ -92,10 +92,11 @@ spec:
       - name: remove-ipfailover
         image: quay.io/openshift/origin-keepalived-ipfailover:{product-version}
         command: ["/var/lib/ipfailover/keepalived/remove-failover.sh"]
-      nodeSelector:
+      nodeSelector: <.>
         kubernetes.io/hostname: <host_name>  <.>
       restartPolicy: Never
 ----
+<.> The `nodeSelector` is likely the same as the selector used in the old IP failover deployment.
 <.> Run the job for each node in your cluster that was configured for IP failover and replace the hostname each time.
```

Comment:
```suggestion
<2> Run the job for each node in your cluster that was configured for IP failover and replace the hostname each time.
```
https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/networking/configuring-ipfailover#nw-ipfailover-remove_configuring-ipfailover

Existing footnote numbering is already not working on the CP, need to update if adding a second, especially.